### DISCUSSION TITLE
3/27(월) 상속

### DISCUSSION THREAD

**Discussion by jjunhaa0211** [Info-iOS/iOStudy#36] [General]
오늘은 상속에 대해서 공부하면 될 것 같습니다. 상속을 하면 여러번의 view를 만들기 않고도 쉽게 커스텀 및 불필요한 코드를 줄일 수 있는 최고의 방법입니다.
상속은 저 같은 경우에는 잘몰랐지만 어느 순산 UIView를 10번 선언하는 저를 보고 상속의 필요성을 느꼈습니다. 때문에 상속에 대해서 빨리 알면 저보다 훨씬 빨리 좋은 코드를 작성할 수 있습니다. 😑

**Comment by Eunho0922** [comment]
넵
